src/components/network/network.js

```diff
@@ -5,37 +5,39 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
 
+import { RemoteRessourceHandler } from '../util/remote-ressource-handler';
+
 const elementIdIndexer = (map, element) => {
     map.set(element.id, element);
     return map;
 };
 
 export default class Network {
-    substations = [];
+    substations;
```

Comment:
why not `= undefined` like this others ?

Reply:
this one is computed from voltages levels